Tests/MusicXMLTests/Simple Types/MIDIDeviceTests.swift

```diff
@@ -0,0 +1,24 @@
+//
```

Comment:
I enclosed this file under `Simple Types`, which is a folder I newly created. lmk if this works. I feel that all types that don't nest other music xml types should be treated as simple types.

Reply:
Works for me.